src/styles/_catalog.scss

```
@@ -1,10 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-items: center;
+  gap: 48px;
+  max-width: 1024px;
   margin: 0 auto;
 
+  @media screen and (max-width: 487px) {
+    & {
+      grid-template-columns: 1fr;
+    }
+  }
```

```suggestion
      grid-template-columns: 1fr;
```
Let it be a default value and when screen become wider we will overwrite it

---

Done